docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,37 @@
+# Grid Layout
+
+## Overview
+
+`GridLayout` is a grid box for the two-dimensional layout. It constraints the x and y position, width, and height of the child actors.
```

Comment:
Kindly avoid hyphenating the words, as recommended in the Google Style Guide: https://developers.google.com/style/hyphens

Reply:
Done.